components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -363,6 +366,8 @@ You need data for all projects.*
 | `run_time_hours` | Runtime value in hours for metrics calculated based on time | `0.000278` |
 | `jobs` | Number of jobs for relevant metrics | `100` |
 
+***Note:** A `0` value for **Data Apps** or **Data Science Sandbox** in `usage_breakdown` indicates a correction. Due to incremental loads, the row couldn’t be deleted, so the original value was set to zero.*
```

Comment:
Sorry, že takhle prudím nad dokumentací, ale možná by bylo lepší mít ty věty identické, než aby se jedna lišila. `indicates` vs `means` a pak `due to incremental` na různých místech.